src/content/content.ts

```diff
@@ -2085,6 +2085,23 @@ export class ContentHandler {
         }
         next = ++next % cycleOrder.length;
       }
+    } else if (dictToShow === 'prev') {
+      dict = this.currentDict;
+
+      let prev =
+        (cycleOrder.length + (cycleOrder.indexOf(this.currentDict) - 1)) %
+        cycleOrder.length;
```

Comment:
If we use the `mod()` function mentioned below I think this becomes just:

```suggestion
      let prev = mod(cycleOrder.indexOf(this.currentDict) - 1, cycleOrder.length);
```